### PR TITLE
Added alert, changed background

### PR DESCRIPTION
**Pull request by pandah3**
Changed alert
Changed background color from red to blue

### CODE REVIEW

File: public/stylesheets/style.css [hunk 1]
```diff
@@ -1,7 +1,7 @@
 body {
   padding: 50px;
   font: 14px "Lucida Grande", Helvetica, Arial, sans-serif;
-  background-color: red;
+  background-color: blue;
```
Comment (1mauricio4) [inline public/stylesheets/style.css]: id be best to have the background color to green

**Review by 1mauricio4 — CHANGES_REQUESTED** [review]

File: public/stylesheets/style.css (hunk 1 above)
Comment (pandah3) [reply public/stylesheets/style.css]: NO

Comment (pandah3) [reply public/stylesheets/style.css]: you spelled it'd wrong

**Review by 1mauricio4 — APPROVED** [review]
this was just great jojo

**Pull request merged**